src/scenes/Home/ISplashPage.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Paper from '@material-ui/core/Paper';
+import ReactMarkdown from 'react-markdown';
+
+/**
+ * @author Reece Dunham
+ */
+export default props => {
+  return (
+    <Paper
+      style={{
```

Comment:
As I mentioned twice previously please use `withStyles`. Check `CirrusCircularProgress.tsx` for an example.